cypress/e2e/fee-fine/input-incorrect-value-to-bill-actual-cost.cy.js

```diff
@@ -0,0 +1,191 @@
+      duration: 6,
+      intervalId: 'Weeks',
+    },
+    feesFinesShallRefunded: {
+      duration: 6,
+      intervalId: 'Months',
+    },
+    patronBilledAfterAgedLost: {
+      duration: 1,
+      intervalId: 'Minutes',
+    },
+    returnedLostItemProcessingFee: false,
+    replacedLostItemProcessingFee: false,
+    replacementProcessingFee: 0.0,
+    replacementAllowed: false,
+    lostItemReturned: 'Charge',
+    id: uuid(),
+  };
+  let itemData;
+
+  before('Create test data', () => {
+    cy.getAdminToken().then(() => {
+      ServicePoints.createViaApi(testData.servicePoint);
+      testData.defaultLocation = Locations.getDefaultLocation({
+        servicePointId: testData.servicePoint.id,
+      }).location;
+      Locations.createViaApi(testData.defaultLocation).then((location) => {
+        InventoryInstances.createFolioInstancesViaApi({
+          folioInstances: testData.folioInstances,
+          location,
+        });
+      });
+
+      cy.createLoanType({
+        name: getTestEntityValue('feeFine'),
+      })
+        .then((loanType) => {
+          testData.loanTypeId = loanType.id;
+        })
+        .then(() => {
+          LostItemFeePolicy.createViaApi(feePolicy);
+          CirculationRules.addRuleViaApi({ t: testData.loanTypeId }, { i: feePolicy.id }).then(
+            (newRule) => {
+              testData.addedRule = newRule;
+            },
+          );
+        });
+      UsersOwners.createViaApi({
+        ...ownerData,
+        servicePointOwner: [
+          {
+            value: testData.servicePoint.id,
+            label: testData.servicePoint.name,
+          },
+        ],
+      }).then((ownerResponse) => {
+        testData.ownerId = ownerResponse.id;
+      });
+      itemData = testData.folioInstances[0];
+    });
+
+    cy.createTempUser([
+      Permissions.uiUserLostItemRequiringActualCost.gui,
+      Permissions.uiUsersDeclareItemLost.gui,
+    ]).then((user) => {
+      testData.userData = user;
+      UserEdit.addServicePointViaApi(
+        testData.servicePoint.id,
+        testData.userData.userId,
+        testData.servicePoint.id,
+      ).then(() => {
+        Checkout.checkoutItemViaApi({
+          itemBarcode: itemData.barcodes[0],
+          userBarcode: testData.userData.barcode,
+          servicePointId: testData.servicePoint.id,
+        });
+        cy.login(testData.userData.username, testData.userData.password, {
+          path: TopMenu.usersPath,
+          waiter: UsersSearchResultsPane.waitLoading,
+        });
+      });
+
+      UsersSearchPane.searchByKeywords(testData.userData.userId);
+      UsersSearchPane.openUser(testData.userData.userId);
+      UsersCard.waitLoading();
+      UsersCard.viewCurrentLoans();
+      UserLoans.openLoanDetails(itemData.barcodes[0]);
+
+      LoanDetails.checkAction(0, 'Checked out');
+      LoanDetails.startDeclareLost();
+      LoanDetails.finishDeclareLost(declareLostComments);
+      LoanDetails.checkAction(0, 'Declared lost');
```

Comment:
Is it possible to do this through the API method?
`UserLoans.declareLoanLostViaApi()`